inst/selenium/src/main/kotlin/com/rstudio/seleniumRunner/Main.kt

```diff
@@ -0,0 +1,79 @@
+package com.rstudio.seleniumRunner
+
+import io.github.bonigarcia.wdm.DriverManagerType
+import io.github.bonigarcia.wdm.WebDriverManager
+import org.openqa.selenium.By
+import org.openqa.selenium.Dimension
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.WebDriverWait
+import kotlin.system.exitProcess
+
+val types = enumValues<DriverManagerType>().map { it.name.toLowerCase() }.joinToString(",")
+
+fun help() {
+    println("""
+            Usage:
+                selenium DRIVER DIMENSIONS URL XPATH TIMEOUT [OPTION]...
+                
+                DRIVER: Name of a Selenium driver. Valid names are: $types
+                DIMENSIONS: Window dimension, in pixels, of the format 1200x800
+                URL: URL to visit.
+                XPATH: XPath selector string. When the selector matches any elements on the page, this tool exits successfully.
+                TIMEOUT: The number of seconds to wait for the selector to match before exiting with an error.
+                OPTION: Zero or more arguments to add as Selenium driver options.
+                
+            Example:
+                java -jar selenium.jar chrome 1200x800 https://news.google.com/ '//body' 30 --headless
+        """.trimIndent())
+    exitProcess(0)
+}
+
+fun driverOptions(driverName: String, args: List<String>): Any? {
+    if (args.isEmpty()) return null
+    val className = when(driverName) {
+        "chrome" -> "org.openqa.selenium.chrome.ChromeOptions"
+        "firefox" -> "org.openqa.selenium.firefox.FirefoxOptions"
+        "opera" -> "org.openqa.selenium.opera.OperaOptions"
+        "iexplorer" -> "org.openqa.selenium.ie.InternetExplorerOptions"
```

Comment:
Edge does not support options?

Reply:
I don't know for sure, but I couldn't find anything like "EdgeOptions" in the Selenium package. phantomjs, selenenium_server_standalone (don't know what that is...) and chromium are the other drivers that didn't seem to have Options classes.

Reply:
`import org.openqa.selenium.edge.EdgeOptions;`

It doesn't support much of anything, but it'd complete the set.